tsdb/functions.go

```diff
@@ -377,11 +379,12 @@ func ReduceCountDistinct(values []interface{}) interface{} {
 	return len(index)
 }
 
-type NumberType int8
```

Comment:
The type is only used for reducing (and only in this package) so, as with interfaceValues, I made it unexported (as I saw no use for it outside the package).

If you want I can just make it exported again and document it.